server/src/test/java/org/apache/druid/query/lookup/LookupReferencesManagerTest.java

```diff
@@ -765,6 +777,91 @@ public void testCoordinatorLookupSync() throws Exception
 
   }
 
+  public static Object[] parametersForTestCoordinatorSelectiveLoadingOfLookups()
+  {
+    return new Object[] {
+        // load all lookups
+        new Object[]{
+            null
+        },
```

Comment:
this can be in a single line without compromising readability, same for the other elements in this array.